Framework/Kernel/src/TimeSeriesProperty.cpp

```diff
@@ -566,6 +572,7 @@ void TimeSeriesProperty<TYPE>::splitByTimeVector(
   std::vector<DateAndTime>::iterator splitter_iter;
   splitter_iter = std::lower_bound(splitter_time_vec.begin(),
                                    splitter_time_vec.end(), tsp_time);
+  // tps_time is start time of time series property
```

Comment:
`tsp_time`

Also, it does not show up until 20 lines further down. Move the comment there?